portal-api/src/scheduler/jobs/send-report-overdue-emails-job/index.test.js

```diff
@@ -0,0 +1,88 @@
+const { subDays } = require('date-fns');
+const { produce } = require('immer');
+const sendReportOverdueEmailsJob = require('.');
+const api = require('../../../v1/api');
+const externalApi = require('../../../external-api/api');
+const sendEmail = require('../../../external-api/send-email');
+const MOCK_BANKS = require('../../../../test-helpers/mock-banks');
+const { EMAIL_TEMPLATE_IDS } = require('../../../constants');
+
+jest.mock('../../../v1/api');
+jest.mock('../../../external-api/bank-holidays');
+jest.mock('../../../external-api/send-email', () => jest.fn());
+
+console.error = jest.fn();
+console.warn = jest.fn();
+console.info = jest.fn();
+
+const originalProcessEnv = process.env;
+
+const sendReportOverdueEmailsJobTask = sendReportOverdueEmailsJob.init().task;
+
+describe('sendReportOverdueEmailsJob', () => {
+  afterEach(() => {
+    process.env = { ...originalProcessEnv };
+    jest.resetAllMocks();
+    jest.useRealTimers();
+  });
+
+  it('does not send emails when report chaser is not due today', async () => {
+    // Arrange
+    process.env.UTILISATION_REPORT_OVERDUE_CHASER_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH = 15;
```

Comment:
It would have effects on other tests, but I have the following the in the `afterEach` function that resets it after each test:
```
process.env = { ...originalProcessEnv };
```

Reply:
Ah very nice :)